src/prefabs/dialog.js

```diff
@@ -0,0 +1,19 @@
+(() => ({
+  name: 'Dialog',
+  icon: 'PanelIcon',
+  category: 'LAYOUT',
+  structure: [
+    {
+      name: 'Dialog',
+      options: [
+        {
+          value: true,
+          label: 'Visible in dev?',
```

Comment:
Users probably don't understand the meaning of `dev`.

Reply:
Oki, I'll rename this to 'Visible' just like in the Alert component :)